app/templates/components/schema-field-radio.hbs

```diff
@@ -1,12 +1,15 @@
 <div class="radio">
   <div class="radio-button-option">
-    {{#radio-button value=false groupValue=value name=key changed="changed"}}
+    {{#radio-button value=false groupValue=value name=key changed="changed" disabled=property.displayProperties.disabled}}
       False
     {{/radio-button}}
   </div>
   <div class="radio-button-option">
-    {{#radio-button value=true groupValue=value name=key changed="changed"}}
+    {{#radio-button value=true groupValue=value name=key changed="changed" disabled=property.displayProperties.disabled}}
       True
     {{/radio-button}}
   </div>
-</div>
\ No newline at end of file
+</div>
+
+Disabled: 
```

Comment:
Seems like this might have been used for debugging purposes, do we want to leave this in?